### PR TITLE
staing/apimachinery: fix typo "perfer" to "prefer"

### PR DESCRIPTION
**Pull request by gyuho**
**What this PR does / why we need it**:

Fix a minor typo.

**Release note**: 

```release-note
"NONE"
```

### CODE REVIEW

**Comment by gyuho** [comment]
@misterikkit Just fixed. Thanks!

File: test/e2e/scheduling/priorities.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		}
 	})
 
-	It("Pod should perfer to scheduled to nodes pod can tolerate", func() {
+	It("Pod should prefer to scheduled to nodes pod can tolerate", func() {
```
Comment (resouer) [inline test/e2e/scheduling/priorities.go]: Have you tried to search perfer globally in the repo? Normally typo is caused by silly copy-paste.

Comment (gyuho) [reply test/e2e/scheduling/priorities.go]: @resouer Found one more, and fixed.

**Comment by resouer** [comment]
Please squash the commits, change looks good.

**Review by resouer — APPROVED** [review]
/lgtm

cc @bsalamat for approve

**Review by misterikkit — APPROVED** [review]
/lgtm

**Review by misterikkit — APPROVED** [review]

**Comment by misterikkit** [comment]
It seems like pull-kubernetes-e2e-gce is flaking here. Can we skip it?

**Review by misterikkit — APPROVED** [review]
/lgtm

**Comment by nikhita** [comment]
/retest

/assign @deads2k 
for approval

**Review by nikhita — APPROVED** [review]
/lgtm

**Review by bsalamat — COMMENTED** [review]
/approve

**Pull request merged**